app/views/upload.scala.html

```diff
@@ -47,6 +48,8 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
                 </button>
             </div>
         </form>
+<!--        Form to redirect user once upload has completed. It sends consignmentId to record processing placeholder page -->
+        @form(routes.RecordsController.recordsPage(consignmentId), Symbol("id") -> "upload-data-form") { }
```

Comment:
Could this be rewritten without the additional mapping, eg: @form(routes.RecordsController.recordsPage(consignmentId)) {}

Reply:
As discussed, this is resolved. As this is needed to find the form with js